[0.086s] Invoking command in '/home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/build/my_mas': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/install/py_pubsub:/home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/install/nodes_comunication:/home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/install/hardware:/opt/ros/humble /usr/bin/cmake --build /home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/build/my_mas -- -j4 -l4
[0.160s] -- Found ament_cmake: 1.3.5 (/opt/ros/humble/share/ament_cmake/cmake)
[0.721s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[0.728s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.740s] -- Found rosidl_adapter: 3.1.5 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.832s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.934s] -- Found geometry_msgs: 4.2.3 (/opt/ros/humble/share/geometry_msgs/cmake)
[1.849s] -- Configuring incomplete, errors occurred!
[1.849s] See also "/home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/build/my_mas/CMakeFiles/CMakeOutput.log".
[1.849s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[1.850s]   execute_process(/usr/bin/python3.10 -m rosidl_adapter --package-name my_mas
[1.850s]   --arguments-file
[1.850s]   /home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/build/my_mas/rosidl_adapter__arguments__my_mas.json
[1.851s]   --output-dir
[1.851s]   /home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/build/my_mas/rosidl_adapter/my_mas
[1.852s]   --output-file
[1.852s]   /home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/build/my_mas/rosidl_adapter/my_mas.idls)
[1.852s]   returned error code 1:
[1.853s] 
[1.854s]   Traceback (most recent call last):
[1.854s] 
[1.855s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[1.855s]       return _run_code(code, main_globals, None,
[1.856s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[1.856s]       exec(code, run_globals)
[1.857s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[1.857s]       sys.exit(main())
[1.860s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[1.860s]       abs_idl_file = convert_to_idl(
[1.861s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 24, in convert_to_idl
[1.861s]       return convert_srv_to_idl(
[1.861s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/srv/__init__.py", line 28, in convert_srv_to_idl
[1.862s]       srv = parse_service_string(package_name, input_file.stem, content)
[1.862s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 859, in parse_service_string
[1.862s]       request_message = parse_message_string(
[1.863s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 544, in parse_message_string
[1.863s]       msg = MessageSpecification(pkg_name, msg_name, fields, constants)
[1.864s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 388, in __init__
[1.865s]       self.base_type = BaseType(
[1.865s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 205, in __init__
[1.866s]       raise InvalidResourceName(
[1.867s] 
[1.868s]   rosidl_adapter.parser.InvalidResourceName: 'Load_file_Request' is an
[1.868s]   invalid message name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$'
[1.869s] 
[1.870s] Call Stack (most recent call first):
[1.870s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[1.871s]   CMakeLists.txt:14 (rosidl_generate_interfaces)
[1.871s] 
[1.872s] [0m
[1.872s] gmake: *** [Makefile:747: cmake_check_build_system] Error 1
[1.873s] Invoked command in '/home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/build/my_mas' returned '2': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/install/py_pubsub:/home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/install/nodes_comunication:/home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/install/hardware:/opt/ros/humble /usr/bin/cmake --build /home/ffelix07/Documents/LabRemo/Orchestrator/ros2_labremoto/build/my_mas -- -j4 -l4
